example.py

```diff
@@ -0,0 +1,25 @@
+import json
+import requests
+
+def revert_risk_api_request(language, revision_id):
```

Comment:
It is a good practice to add [type hints](https://docs.python.org/3/library/typing.html) so that the use knows how to use the function without having to read the code.
```suggestion
def revert_risk_api_request(language: str, revision_id: int):
```